src/components/StyledList/StyledList.mdx

```diff
@@ -0,0 +1,100 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as StyledListStories from "./StyledList.stories";
+
+<Meta of={StyledListStories} />
+
+# StyledList
+
+| Component Version | DS Version   |
+| ----------------- | ------------ |
+| Added             | `1.3.0`      |
+| Latest            | `Prerelease` |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [StyledList Children](#styledlist-children)
+- [Font Size](#font-size)
+
+## Overview
+
+<Description of={StyledListStories} />
+
+## Component Props
+
+<Canvas of={StyledListStories.WithControls} />
+
+<ArgTypes of={StyledListStories.WithControls} />
+
+## Accessibility
+
+Internally, the `StyledList` component implements native HTML markup for
+unordered lists. When data is passed into the `listItems` data prop, the
+appropriate children `li` HTML elements are used.
+
+Resources:
+
+- [W3C WAI Content Structure Lists](https://www.w3.org/WAI/tutorials/page-structure/content/#lists)
+- [Deque University Appropriate Semantic Markup for Lists](https://dequeuniversity.com/tips/list-markup)
+
+## StyledList Children
+
+### li children
+
+Pass in `<li>` children elements as you normally would for the `<ul`> element.
+
+<Source
+  code={`
+// With \`li\` elements
+<StyledList>
+  <li>
+    FULL TITLE: <i>"...and for the <b>City of New York</b>"</i>
+  </li>
+  <li>
+    ALTERNATIVE TITLES: <i>"...Society of <b>New York</b> delivered at..."</i> + 1 more
+  </li>
+  <li>SUBJECTS: <i>"...<b>City of New York</b>, in the"</i> + 2 more</li>
+</StyledList>
+`}
+  language="jsx"
+/>
+
+<Canvas of={StyledListStories.LiChildren} />
```

Comment:
What do you think about removing this and the `listItems` prop stories?  Don't remove the docs.  Just remove the rendered story.  And then move the `Font Size` section above the `StyledList Children` section.

Reply:
Why this change? This means removing the rendered lists but still keep the code examples? Storybook doesn't have an easy way to not display a story in the sidebar when a story is exported in the `.stories.tsx` file.

But did move `Font Size` higher up.

Reply:
Those two stories on their own, without the associated docs, don't really mean much, so I think it would best to just we removed them from the sidebar.  But if there is no way to render the components on the docs page without also listing them in the sidebar, then leave it as is.  Just update the order of the stories in the stories file.  Right now the sidebar is not showing the same order as the Docs page.